functions/ecs-shutdown-services/README.md

```diff
@@ -0,0 +1,16 @@
+# ECS Shutdown Services
+
+## Overview
+This function can be used to gracefully shutdown all application workloads running for your provisioned tenants. It does this by setting the `desiredCount` attribute of the ECS service for each tenant to zero (0). [Watch a detailed walkthru](https://www.twitch.tv/videos/1065389231) of building this solution during our [Office Hours](https://github.com/awslabs/aws-saas-boost/discussions/106).
+
+## Why would you turn off your SaaS application?
+Excellent question! You should **not** use this function for your production environments. Your SaaS customers expect your service to be available at all times. However, for development and other non-production environments, it may be useful to temporarily shutdown your application tasks as a way to save operational costs. ECS tasks that run on Fargate are only billed when they are running. If you are running your tasks on EC2, you will still incur the costs of the EC2 instances in your ECS cluster.
+
+## How do I use it?
```

Comment:
The example `enable.sh` and `disable.sh` scripts fail if the lambdas have not been set up. Should the READMEs include some instructions for updating an existing SaaS Boost installation to include these Lambdas?

Reply:
Fair question. We don't include specific instructions on how to pull/merge with any other commits. The pattern is always to run the installer and choose update existing install. That will take care of everything.

Reply:
And, to be clear, do the shell scripts crash or do they give you sensible output and exit?

Reply:
Once again, maybe the best option here is to suggest running the installer if we find that the lambdas haven't been installed. If we can lower the bar for entry for a new contributor, we should.